CkCreative/fff
# Merge dev to release

#### What does this PR do?
Add API versioning 
#### How should this be manually tested?

- Install Git
- Install Python 3
- Install `pytest` in your virtualenv
- Open console and run:
> `git clone https://github.com/CkCreative/fff.git`
>`cd fff && git checkout dev`
>`python run.py`
- On a new console window, cd to the /fff project directory and run:
>`pytest tests.py`
- You can also use Postman to test the endpoints. 

> Sample data to post with **POST** `/api/v1/orders`: `{
        "id" : 11,
        "customer_id" : 1254,
        "status" : ""
}`

> Sample data to update order status using **PUT** `/api/v1/orders/<orderID>`:
`{
        "status" : "Completed"
 }`

> **Only valid JSON is accepted for POST and PUT**
> To be able to test **PUT**, first **POST** and then try **PUT** to see the changes take effect.

#### What are the relevant Pivotal Tracker stries?
**Chore**: Add API versioning [160314162](https://www.pivotaltracker.com/story/show/160314162)
